### DISCUSSION TITLE
User options submenu menu item announcements

### DISCUSSION THREAD

**Discussion by rhoadsstevens** [ManifoldScholar/manifold#3455] [Accessibility]
Priority level 1 (out of 3) from UW Accessible Technologies

WCAG mapping: 1.3.1 Info and Relationships

STEPS TO REPRODUCE:
1. With a screen reader, navigate to the User Options menu button
2. Press space to open the submenu and then tab into the submenu
3. In interactive mode, tab to read the menu items in the submenu
4. Press ESC to close the submenu or navigate to the "Close Panel" button and press space/enter

OBSERVED BEHAVIOR:
Submenu items are not announced with their corresponding index. For instance, the first menu item is announced as "edit profile" instead of "edit profile 1 of 4". Additionally, upon closing the submenu, the submenu is not announced as collapsed and using the "Close panel" button keeps your focus on the "User Profile" button but there is no longer a focus indicator.

EXPECTED BEHAVIOR: 
Having the menu indexes announced informs the user of how long the submenu is and communicates where they are in the list (1 of 100 vs 99 of 100). Additionally, dynamic visual changes should be announced to screen reader users. Thus, when a submenu has collapsed, this should be announced to users.

**Comment by tsmyre** [comment]
**CIC Response**
Marking as “won’t fix” since UW indicates that markup and ARIA attributes are correct
